src/Lua/sys_base_objects_description.lua

```diff
@@ -416,7 +416,9 @@ return
                 LS_down = { name = "Датчик нижнего уровня", defaultValue = "LS1" },
                 LT = { name = "Датчик текущего уровня", defaultValue = "LT1" },
                 TE = { name = "Датчик температуры", defaultValue = "TE1" },
-                out_pump = { name = "Откачивающий насос", defaultValue = "M1" },
+                -- out_pump defaultValue пустое т.к по другому происходит
+                -- обработка ОУ. Обрабатывается не объект, а устройство.
+                out_pump = { name = "Откачивающий насос", defaultValue = "" },
```

Comment:
Может такой вариант:
```suggestion
                out_pump = { name = "Откачивающий насос" },
```

Reply:
Да, ты прав. Не досмотрел